pkgs/build-support/fetchnextcloudapp/default.nix

```
@@ -8,19 +8,12 @@
 , patches ? [ ]
 , description ? null
 , homepage ? null
+, unpack ? true # whether to use fetchzip rather than fetchurl
```

Note that this is a non-breaking change until we set
```suggestion
, unpack ? false # whether to use fetchzip rather than fetchurl
```
by default. I suggest we do so, accompanied by release notes, before NixOS 24.11.

---

Sounds good :+1: 
Do you want to file a PR for that?